### PR TITLE
[14.0][IMP] stock_quant_manual_assign: make several columns sortable in wizard

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
We want to be able to sort by some columns in the wizard. Only stored columns are sortable.

### CODE REVIEW

**Review by yostashiro — APPROVED** [review]
LGTM

File: stock_quant_manual_assign/wizard/assign_manual_quants.py [hunk 1]
```diff
@@ -116,6 +116,7 @@ class AssignManualQuantsLines(models.TransientModel):
     _name = "assign.manual.quants.lines"
     _description = "Assign Manual Quants Lines"
     _rec_name = "quant_id"
+    _order = "lot_id, location_id, package_id"
```
Comment (pedrobaeza) [inline stock_quant_manual_assign/wizard/assign_manual_quants.py]: I think you should sort first by location

Comment (MiquelRForgeFlow) [reply stock_quant_manual_assign/wizard/assign_manual_quants.py]: Done :)

**Review by pedrobaeza — APPROVED** [review]

**Comment by MiquelRForgeFlow** [comment]
Forward ported in https://github.com/OCA/stock-logistics-warehouse/pull/1445.

**Pull request merged**